test/ingress/aws/aws_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build aws
+// +build e2e
+
+package aws
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/kinvolk/lokomotive/pkg/util/retryutil"
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	retryIntervalSeconds = 5
+	maxRetries           = 60
+)
+
+func TestAWSIngress(t *testing.T) {
+	client := testutil.CreateKubeClient(t)
+
+	i, err := client.ExtensionsV1beta1().Ingresses("httpbin").Get("httpbin", metav1.GetOptions{})
+	if err != nil {
+		t.Fatalf("getting httpbin ingress: %v", err)
+	}
+
+	h := i.Spec.Rules[0].Host
+
+	err = retryutil.Retry(retryIntervalSeconds*time.Second, maxRetries, func() (bool, error) {
+		resp, err := http.Get(fmt.Sprintf("https://%s/get", h))
```

Comment:
My rationale was to cover as much critical parts as I can with this simple test. I can definitely split it into an HTTP and an HTTPS test if that makes more sense to you.

To emphasize, increasing test coverage is not the scope of this PR. I've added this test following @invidian's request and would like to avoid delaying the merge further if we can since **things are currently broken on master**.

Reply:
I'm totally fine with this test and the approach in general. e2e test don't need to be very specific, we can debug de failure when introduced :)